src/platform/datapath_raw_xdp.c

```diff
@@ -1570,6 +1560,26 @@ CxPlatXdpTx(
     }
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+void
+CxPlatDpRawTxEnqueue(
+    _In_ CXPLAT_SEND_DATA* SendData
+    )
+{
+    XDP_TX_PACKET* Packet = (XDP_TX_PACKET*)SendData;
+    XDP_QUEUE* Queue = Packet->Queue;
+    if (Queue->Worker->ProcIndex == SendData->ProcIndex) {
+        CxPlatListInsertTail(&Queue->WorkerTxQueue, &Packet->Link);
```

Comment:
You're introducing reordering by putting this into the `WorkerTxQueue` directly.

Reply:
Actually, never mind.